core/trino-main/src/main/java/io/trino/server/security/InsecureAuthenticator.java

```diff
@@ -70,7 +70,7 @@ public Identity authenticate(ContainerRequestContext request)
         }
 
         if (user == null) {
-            throw new AuthenticationException("Basic authentication or " + TRINO_HEADERS.requestUser() + " must be sent", BasicAuthCredentials.AUTHENTICATE_HEADER);
+            throw new AuthenticationException("Authentication credentials or " + TRINO_HEADERS.requestUser() + " must be sent", BasicAuthCredentials.AUTHENTICATE_HEADER);
```

Comment:
This seems wrong. `InsecureAuthenticator` is only used over non-secure connections, which don't allow authentication. `InsecureAuthenticator` accepts basic authentication without a password for the purpose of providing the username.

Reply:
This change seems unrelated to the JDBC driver change. If you feel strongly that this should be changed, please create a new PR with a description explaining why.

Reply:
I see, thanks, I totally misinterpreted the error message. I've dropped this commit.